packages/design-system/src/components/server/card.tsx

```diff
@@ -0,0 +1,34 @@
+import { twMerge } from "@repo/design-system/utils";
+import { type VariantProps, cva } from "class-variance-authority";
+
+export type Card = {
+  children: React.ReactNode;
+} & VariantProps<typeof CardVariants>;
+
+const CardVariants = cva("", {
+  variants: {
+    color: {
+      white: "ko:bg-white",
+      transparent: "ko:bg-transparent",
+    },
+    corner: { topRight: "ko:rounded-l ko:rounded-br", topLeft: "ko:rounded-r ko:rounded-bl", bottomRight: "ko:rounded-t ko:rounded-bl", bottomLeft: "ko:rounded-t ko:rounded-br" },
```

Comment:
Just a little tweak: we can do `ko:rounded` for all variants and then just `ko:rounded-l-none` etc. for the sharp corners.